lib/cypress/app_config.rb

```diff
@@ -40,5 +40,12 @@ def self.[]=(key, val)
       # Allow setting app config during app init, before Rails.cache exists
       clear
     end
+
+    def self.load_yml
+      app_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/cypress.yml")).result)
+      names_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/names.yml")).result)
+      app_config['randomization']['names'] = names_config['names']
```

Comment:
So the reason we have it as is is because the names were originally part of the "randomization" of AppConfig. In breaking it up I wanted to bring it back to where it originally was once imported. However, if you think it would be neater to do it with a new constant (maybe NAMES_RANDOM), then let me know and I will get on that.